jenkins/attach_agent.py

```diff
@@ -14,10 +14,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""Attach Jenkins agents."""
 
 import ConfigParser
 import sys
 
+# pylint: disable=import-error
```

Comment:
a) Can you put the disable just on the `from jenkinsapi` line instead of the whole file?
b) Can you check with the rest of our team (Joe/Ryan/Cole) about how to fix this?